app/validators/sections/section_validator.py

```diff
@@ -110,27 +94,19 @@ def validate_blocks(self, group_id):
 
     def validate_routing(self, schema_element, group):
         if "routing_rules" in schema_element:
-            if any("goto" in rule for rule in schema_element["routing_rules"]):
-                routing_validator = RoutingValidator(
-                    schema_element, group, self.questionnaire_schema
-                )
-            else:
-                routing_validator = NewRoutingValidator(
-                    routing_rules=schema_element["routing_rules"],
-                    group=group,
-                    origin_id=schema_element["id"],
-                    questionnaire_schema=self.questionnaire_schema,
-                )
+            routing_validator = RoutingValidator(
+                routing_rules=schema_element["routing_rules"],
+                group=group,
+                origin_id=schema_element["id"],
+                questionnaire_schema=self.questionnaire_schema,
+            )
             self.errors += routing_validator.validate()
-        if "skip_conditions" in schema_element:
-            if isinstance(schema_element["skip_conditions"], list):
-                self.validate_skip_conditions(
-                    schema_element["skip_conditions"], schema_element["id"]
-                )
-            elif isinstance(schema_element["skip_conditions"], dict):
-                self.validate_new_skip_conditions(
-                    schema_element["skip_conditions"], schema_element["id"]
-                )
+        if "skip_conditions" in schema_element and isinstance(
+            schema_element["skip_conditions"], dict
+        ):
```

Comment:
Is the isinstance check needed anymore?.

Reply:
Now only doing:

```python
if "skip_conditions" in schema_element:
```